cmake/modules/AddSwift.cmake

```diff
@@ -359,7 +359,7 @@ function(_add_variant_link_flags)
     endif()
   elseif("${LFLAGS_SDK}" STREQUAL "ANDROID")
     list(APPEND result
-        "-ldl" "-llog" "-latomic"
+        "-ldl" "-llog" "-latomic" "-licudata" "-licui18n" "-licuuc"
```

Comment:
I tried it with this order: "-ldl" "-llog" "-latomic" "-licui18n" "-licudata" "-licuuc"
$ adb shell LD_LIBRARY_PATH=/data/local/tmp /data/local/tmp/hello
And I got: dlopen() failed on `/data/local/tmp/hello': dlopen failed: Aborted